lib/wibox/layout/fixed.lua

```diff
@@ -110,6 +110,14 @@ function fixed:get_children()
     return self.widgets
 end
 
+--- Replace the layout children
+-- @tparam table children A table composed of valid widgets
+function fixed:set_children(children)
+    if not children then return self:reset() end
```

Comment:
Sorry, but

>  "No!".

I think it make the API worst and given (my) the use case of `get_children`, it make no sense

> Well, is it really nice?

I think so, I see myself wanting this sort of thing to make my life easier. It does what it claim to do, the the children "to nothing"